astroplan/observer.py

```diff
@@ -1620,3 +1620,73 @@ def is_night(self, time, horizon=0*u.deg, obswl=None):
 
         solar_altitude = self.altaz(time, target=get_sun(time), obswl=obswl).alt
         return bool(solar_altitude < horizon)
+
+    def local_sidereal_time(self, time, kind='apparent', model=None):
+        """
+        Convert ``time`` to local sidereal time for observer.
+
+        This is a thin wrapper around the `~astropy.time.Time.sidereal_time`
+        method.
+
+        Parameters
+        ----------
+        time : `~astropy.time.Time` or other (see below)
+            Time of observation. This will be passed in as the first argument to
+            the `~astropy.time.Time` initializer, so it can be anything that
+            `~astropy.time.Time` will accept (including a `~astropy.time.Time`
+            object)
+
+        kind : {'mean', 'apparent'} (optional)
+            Passed to the ``kind`` argument of
+            `~astropy.time.Time.sidereal_time`
+
+        model : str or `None`; optional
+            Passed to the ``model`` argument of
```

Comment:
Maybe just add something like ``The precession/nutation model to assume - see `~astropy.time.Time.sidereal_time` for more details.`` ?  That way the user gets at least some hint of what sort of "model" it is.